wal.proto

```
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package wal;
```

Buf (1.47.2)

Files with package "wal" must be within a directory "wal" relative to root but were in directory ".".

(PACKAGE_DIRECTORY_MATCH)